### PR TITLE
docs: add charts page

### PR DESCRIPTION
**Pull request by amalitsky**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [X] Documentation content changes
- [X] clarity.design website / infrastructure changes

## What is the current behavior?

- no documentation for using clarity with charts 
- Issue Number: N/A

## What is the new behavior?
- adds _Charts_ page under _Foundation_

## Does this PR introduce a breaking change?
- [X] No

## Other information
- work in progress

highcharts-textures-switch

### CODE REVIEW

**Comment by gnomeontherun** [comment]
I changed the branch target to `topics/chart` so we can refine the content before release but also asking @mathisscott and @hippee-lee to help with a review next week so we can close this out in the interim.

File: apps/website/foundation/charts/README.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+<br/>
+
+1. Color accessibility derives from adjacent contrast. Two colors may have sufficient contrast (3.1:1) when
+   measured against the background, but they may also be below the threshold relative to one another.
+   Using high contrast separations between adjacent colors (such as black or white borders) will often solve this.
+2. Textures provide a method of differentiating one area from the next which is not dependent on color.
+   Consider if using texture for your charts is appropriate. Clarity provides the following textures - remember that
+   the texture itself must be of sufficient contrast (3.1:1) against its background in order to be accessible.
+3. Direct labeling. Accessibility is improved by maintaining close proximity between labels and the item they refer to.
+   Legends can be less useful particularly in magnification use. In the example below the correspondence between blue in
```
Comment (astorije) [inline apps/website/foundation/charts/README.md:232]: Do you have screenshots for those left/right examples?

Comment (amalitsky) [reply apps/website/foundation/charts/README.md:232]: Jeremie, live demo is available here: https://6025b50934b3380a109440e6--vmware-clarity.netlify.app/foundation/charts/

Comment (astorije) [reply apps/website/foundation/charts/README.md:232]: Thanks @amalitsky, I have indeed been using that preview, it's very useful and helpful!
However, the examples you mention there seem missing:

Screen Shot 2021-02-24 at 10 09 39 PM

Comment (amalitsky) [reply apps/website/foundation/charts/README.md:232]: Got the question now, thanks.

There was no illustration originally planned for the highlighted section.

That said, content is not in it's final form at this point. Meaning both - text as well as illustrations might change based on the review and content provided by the design team.

**Review by hippee-lee — APPROVED** [review]

**Review by gnomeontherun — APPROVED** [review]
I'm approving this to our topic branch, and we'll queue up a task to complete the content.

**Pull request merged**